src/themes/dinside/index.js

```diff
@@ -5,7 +5,7 @@ import colors from './colors';
 const variables = {
 	mainFont: "'Open Sans', helvetica, arial, sans-serif",
 	headingsFont: "'Open Sans', helvetica, arial, sans-serif",
-	headingsWeight: 800,
+	headingsWeight: 700,
```

Comment:
Did a designer ask you to do this? I see that Dinside generally uses 800 for headings in other systems.

Reply:
![Screen Shot 2019-03-21 at 11 29 00 AM](https://user-images.githubusercontent.com/26900246/54743442-d1fde600-4bcc-11e9-9408-885a7c081b56.png)
confirmation from Designer

Reply:
Spoke to Liselotte now. We agreed to use 800 in order to have the same headings as on the front page and the single article pages.